cmd/expiration-mailer/main.go

```diff
@@ -88,9 +147,17 @@ func (m *mailer) sendNags(conn bmail.Conn, contacts []string, certs []*x509.Cert
 			m.log.AuditErrf("parsing contact email %s: %s", contact, err)
 			continue
 		}
-		if parsed.Scheme == "mailto" {
-			emails = append(emails, parsed.Opaque)
+		if parsed.Scheme != "mailto" {
+			continue
+		}
+		address := parsed.Opaque
+		err = m.addressLimiter.check(address)
+		if err != nil {
+			m.log.AuditInfof("not sending mail: %s", err)
```

Comment:
I don't think this or the log line on 147 above needs to be an audit logged. We
run the expiry notification service as best effort.

Reply:
oops, good catch! I meant this to be an Infof. Up above on line 147, I changed to an Errf.